app/common/application/resources/app.json

```diff
@@ -2,9 +2,9 @@
   "applications": [
       {"name":"APP_REGISTRATION", "url":"/registration", "logo":"images/hospital.png"},
       {"name":"APP_CLINIC", "url":"/clinic", "logo":"images/clinic.png"},
-      {"name":"APP_VITALS", "url":"/vitals", "logo":"images/vitals.png"},
+      {"name":"APP_VITALS", "url":"/vitals", "logo":"images/vitals.png"}/*,
```

Comment:
Hi @ningosi I suggest that here be added an attribute `active: true/false` for all the app objects. This way we can simply change this config data to indicate those apps that will be visible in the dashboard, adding more configurability, without having to comment or remove them. Something like:
```
[
...
{"name":"APP_REGISTRATION", "url":"/registration", "logo":"images/hospital.png", "active": "false"},
...
]
```